web/ui/react-app/src/Graph.tsx

```diff
@@ -19,6 +19,15 @@ function getGraphID() {
   return graphID++;
 }
 
+export type GraphSeriesValue = number | null;
+
+export interface GraphSeriesXY {
```

Comment:
I think the `XY` part of the name is unnecessary, maybe just `GraphSeries`?

Reply:
Ok, I'll rename it.